### PR TITLE
Add SQL support section to Accumulo connector

### PR DESCRIPTION
**Pull request by Jessie212**

### CODE REVIEW

File: docs/src/main/sphinx/connector/accumulo.rst [hunk 1]
```diff
@@ -698,3 +698,20 @@ the output of the ``DESCRIBE`` statement.
...
+the Accumulo database. In addition to the :ref:`globally available
+<sql-globally-available>` and :ref:`read operation <sql-read-operations>`
+statements, the connector supports the following features:
+
+* :doc:`/sql/insert`
+* :doc:`/sql/create-table`
+* :doc:`/sql/create-table-as`
+* :doc:`/sql/drop-table`
+* :doc:`/sql/create-schema`
+* :doc:`/sql/drop-schema`
```
Comment (ebyhr) [inline docs/src/main/sphinx/connector/accumulo.rst:717]: Remove create-schema & drop-schema as unsupported.

**Review by rosewms — APPROVED** [review]
Approved, pending the removal of unsupported info.

**Comment by jhlodin** [comment]
Please delete the whole "Unsupported features" section starting at line 66, since it essentially just says "ALTER TABLE and DELETE are not supported" which is redundant with the information you're adding.

**Review by ebyhr — APPROVED** [review]

**Comment by ebyhr** [comment]
Merged, thanks!

**Pull request merged**